var/spack/repos/builtin/packages/intel-parallel-studio/package.py

```diff
@@ -120,12 +120,7 @@ def blas_libs(self):
             root=join_path(self.prefix, 'mkl', 'lib', 'intel64'),
             shared=shared
         )
-        system_libs = [
-            'libpthread.{0}'.format(suffix),
-            'libm.{0}'.format(suffix),
-            'libdl.{0}'.format(suffix)
-        ]
-        return mkl_libs + system_libs
```

Comment:
Does anyone know why we were doing this? It breaks `hpl` because we aren't providing the full path to the libraries.